src/main/scala/latis/input/JsonArrayAdapter.scala

```diff
@@ -0,0 +1,30 @@
+package latis.input
+
+import latis.model.DataType
+
+/**
+ * Adapter for parsing HAPI JSON data.
+ */
+class JsonArrayAdapter(model: DataType)
+  extends TextAdapter(model, new TextAdapter.Config("dataMarker" -> "\"data\":\\[")) {
```

Comment:
I'm not convinced this is general enough to call it `JsonArrayAdapter`. I'd call it `HapiJsonArrayAdapter`. This class doesn't take a config in its constructor, so if I understand correctly, you can't overwrite the `dataMarker` value from `"data":[`. It's not guaranteed that will be the data marker for all JSON array datasets.

Reply:
It would need a little evolution but the bulk of the ability to parse an array of arrays will be handy.

Reply:
Okie doke.